contracts/MarginCalculator.sol

```diff
@@ -138,15 +190,15 @@ contract MarginCalculator {
                 marginRequired = _getCallSpreadMarginRequired(shortAmount, longAmount, shortStrike, longStrike);
             }
         } else {
-            FPI.FixedPointInt memory shortCashValue = _uint256ToFPI(getExpiredCashValue(address(short)));
+            FPI.FixedPointInt memory shortCashValue = _uint256ToFPI(_getExpiredCashValue(address(short)));
             FPI.FixedPointInt memory longCashValue = hasLongInVault
-                ? _uint256ToFPI(getExpiredCashValue(_vault.longOtokens[0]))
+                ? _uint256ToFPI(_getExpiredCashValue(_vault.longOtokens[0]))
```

Comment:
shouldn't the getExcessCollateral Function return the value in the collateral asset? So shouldn't this call getExpiredPayoutRate?

Reply:
no, all the formula we had were based on cash value in strike, and the getExpiredCashValue function remains the same